src/topic.ts

```diff
@@ -35,7 +38,7 @@ export class Topic<T extends TigrisTopicType> {
 		return this._topicName;
 	}
 
-	publishMany(_options?: PublishOptions, ...messages: Array<T>): Promise<Array<T>> {
+	publishMany(options?: PublishOptions, ...messages: Array<T>): Promise<Array<T>> {
```

Comment:
Should we have publish that accepts a single message then we can have this Api with first parameter as an array and second as the options. Otherwise user need to pass null or empty options even if they don't need option. @JigarJoshi @mattayres

Reply:
Yeah. Appears @JigarJoshi already did this for `insertMany` too.